subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/install/internal/AdoptOpenJdkRemoteBinary.java

```diff
@@ -111,7 +111,7 @@ private String determineReleaseState() {
     }
 
     private String getServerBaseUri() {
-        String baseUri = System.getProperty("org.gradle.jvm.toolchain.install.internal.adoptopenjdk.baseUri", "https://api.adoptopenjdk.net/");
+        String baseUri = System.getProperty("org.gradle.jvm.toolchain.install.adoptopenjdk.baseUri", "https://api.adoptopenjdk.net/");
```

Comment:
Clicked approve too fast. I once again am not sure if we should use a system property or a Gradle property.
For me setting this to a different value belongs to the `~/.gradle/gradle.properties` or the equivalent in a Gradle custom distribution.
Makes sense?